FIAT/functional.py

```diff
@@ -71,39 +71,18 @@ def __init__(self, ref_el, target_shape,
         return
 
     def evaluate(self, f):
-        """Evaluates the functional on some callable object f."""
-        result = 0
-
-        # non-derivative part
```

Comment:
Was this code logically broken, i.e. broken for reasons other than having the ScientificPython import commented out?